common/oauth2-proxy/components/istio-m2m/requestauthentication.yaml

```
@@ -4,11 +4,39 @@ metadata:
   name: m2m-token-issuer
   namespace: istio-system
 spec:
+  # we only apply to the ingress-gateway because:
+  #  - there is no need to verify the same tokens at each sidecar
+  #  - having no selector will apply to the RequestAuthentication to ALL
```

I think the "to" is too much

---

I am going to assume you mean there is a typo because I included an extra `to` word, it should be:

```suggestion
  #  - having no selector will apply the RequestAuthentication to ALL
```

---

It seems to commiting your suggestion here breaks the DCO because it does not properly sign. Feel free to just overwrite it and commit yourself.